content/_index.md

```diff
@@ -1,19 +1,45 @@
 +++
-title = "Snowplow Consent Accelerator"
+title = "Snowplow Consent Tracking Accelerator"
 menuTitle="Introduction"
 chapter = false
 weight = 1
 +++
 
-Welcome to the **Consent Tracking Plugin** accelerator. This accelerator will help you create data from your Consent Management Platform using the enhanced consent plugin from Snowplow. Once completed, you will have a consent health check dashboard providing insight into the consent status of your users.
+### Welcome to the Consent Tracking accelerator
+
+This accelerator will help you collect data from user interactions with your Consent Management Platform _(CMP)_ and their preferences, using the enhanced consent plugin from Snowplow.
+
+Data collection and user consent has become a main concern when talking about product, personalisation, marketing and ML-driven actions. At the same time, CMPs can provide a limited and siloed amount of data about user preferences and individual selection.
```

Comment:
```suggestion
User consent for data collection is vital when talking about product, personalisation, marketing and ML-driven actions. CMPs provide a limited amount of data about user preferences but with the enhanced consent plugin this can become a rich source of information to understand consent in your business.
```